### DISCUSSION TITLE
Full and Partial keep days

### DISCUSSION THREAD

**Discussion by poudenes** [jcwillox/hass-auto-backup#92] [Q&A] [answered]
Hi All,

I created 2 automations. 1 for a full backup on Sunday and 1 partial backup everyday they without Sunday.
For full backup keep_days: 1 (so every new backup the old one will removed)
For Partial backup keep_days: 6 

But will this work? Or will the full backup also delete the partial backups? (both on same location) because later I use Rsync add-on to sync them to my Synology nas.

**Comment by poudenes** [comment]
I already found way to backup them into 2 separate folders and rsync them into their own folders on Minio.
Now the keep _days will work specific for the backup that is used.

**Comment by jcwillox** [comment, marked as answer]
Yeah every backup stores it's expiration time individually so only the ones that have past their keep days will be deleted. I use a similar solution with the automatic backups blueprint from the wiki and the rclone add-on, it creates hourly, daily, weekly backups and syncs all except the hourly ones, when auto backup deletes an expired backup it is deleted from the cloud as well. The way I differentiate which ones are backed up are via the name.

**Reply by poudenes** [reply]
> Yeah every backup stores it's expiration time individually so only the ones that have past their keep days will be deleted. I use a similar solution with the automatic backups blueprint from the wiki and the rclone add-on, it creates hourly, daily, weekly backups and syncs all except the hourly ones, when auto backup deletes an expired backup it is deleted from the cloud as well. The way I differentiate which ones are backed up are via the name.

So you can have all backups in 1 folder? Rclone will only delete the backups that belong to the automation or check them because of the name?

**Reply by jcwillox** [reply]
In my case rclone isn't doing the deletion auto backup does, rclone will sync the backups matched by name and the sync command will delete backups in the remote that no longer exist locally.
